pkg/manager/manager_tasks.go

```diff
@@ -166,6 +166,7 @@ func (s *singleton) Generate(sprites bool, previews bool, markers bool, transcod
 		}
 		totalsNeeded := s.neededGenerate(scenes, sprites, previews, markers, transcodes)
 		if totalsNeeded == nil {
+			logger.Infof("Counting content timeout!")
```

Comment:
Is this supposed to be here?

Reply:
Yes. If the timeout occurs i thought we should be a little more verbose than just printing the generating content message. It probably isn't necessary  but thats where it should be if we decide to keep it.

Reply:
Yeah, I get it now. I'd probably change the phrasing. `Taking too long to count content. Skipping...`, or something like that.

Reply:
changed.